client/disputes/index.tsx

```diff
@@ -207,14 +207,17 @@ export const DisputesList = (): JSX.Element => {
 	const rows = disputes.map( ( dispute ) => {
 		const clickable = ( children: React.ReactNode ): JSX.Element => (
 			<ClickableCell
-				href={ getDetailsURL( dispute.dispute_id, 'disputes' ) }
+				href={ getDetailsURL( dispute.charge_id, 'transactions' ) }
 			>
 				{ children }
 			</ClickableCell>
 		);
 
 		const detailsLink = (
-			<DetailsLink id={ dispute.dispute_id } parentSegment="disputes" />
+			<DetailsLink
+				id={ dispute.charge_id }
+				parentSegment="transactions"
+			/>
```

Comment:
Aha I see… IMO `DetailsLink` makes this PR harder to review. I was expecting the PR be trivial tweaks to URLs, but these URLs are abstracted away by the unusual (IMO) interface.

Reply:
TLDR I need to understand how `DetailsLink` works, and what `parentSegment` means to review. Feels convoluted 😁

Reply:
I will remove all changes to `DetailsLink` from this PR, it is not in scope (more related to #7363 if anything).

Reply:
Removed changes to `DetailsLink` in 2e47bb83b8d0a85deecfb658622d29be1b8b44eb.